client/src/components/eventsPanel.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
```

Comment:
It looks like this PR is treating `eventsPanel.tsx` as a new file instead of a rename from `eventsPanel.js`. Is that what you intended?

Reply:
It isn't, going to try to get that fixed.